expan/core/experiment.py

```diff
@@ -399,9 +400,14 @@ def delta(self, method='fixed_horizon', kpi_subset=None, derived_kpis=None, **kw
 		else:
 			m = method_table[method]
 			f, a1, a2 = m[0], m[1], m[2]
-			return f(a1, a2, **kwargs)
+			if method == 'fixed_horizon':
+				# pass initialized results object into fixed_horizon_delta
+				return f(res, a1, a2, **kwargs)
+			else:
+				return f(a1, a2, **kwargs)
 
-	def fixed_horizon_delta(self, 
+	def fixed_horizon_delta(self,
```

Comment:
Is introducing an additional parameter to fixed_horizon_delta() worth it? It seems to me that this introduces an inhomogeneity across the fourth `delta` workers